mongo/tests/test_e2e.py

```diff
@@ -45,7 +45,11 @@
     'mongodb.stats.objects',
 ]
 
-MONGOD_METRICS = BASE_METRICS + ['mongodb.fsynclocked']
+MONGOD_METRICS = BASE_METRICS + [
+    'mongodb.metrics.queryexecutor.scannedps',
```

Comment:
This one was documented but not collected?

Reply:
You can add an aggregator.assert_all_metrics_covered() to ensure all metrics are asserted, although I suggest doing that in a separate PR in case you find gremlins under a rock